.github/workflows/semconvgen.yml

```
@@ -68,3 +68,11 @@ jobs:
         else
           tag_and_push "${GITHUB_REF#"refs/tags/"}"
         fi
+    - name: Push the Dev Docker image
+      if: startsWith(github.ref, 'refs/pull/243')
```

```suggestion
      if: startsWith(github.ref, 'refs/heads/feature/')
```
If you target your PR at https://github.com/open-telemetry/build-tools/tree/feature/codegen-by-namespace